keyboards/nack/readme.md

```diff
@@ -0,0 +1,16 @@
+# nack
+
+
+A ISO-friendly ortholinear keyboard, 13x4 keys.  
+See [screenshots](https://imgur.com/a/PPO18Lc).
+
+* Keyboard Maintainer: [pastapojken](https://github.com/pastapojken)
+* Hardware Supported: nack PCB
+* Hardware Availability: [github repo](https://github.com/pastapojken/nack)
+
+Make example for this keyboard (after setting up your build environment):
+
```

Comment:
```suggestion

    make nack:default
    
Flashing example for this keyboard:
```

Reply:
Can you also add "clear instructions on how to reset the board into bootloader mode" as per [the PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)